modules/apps/application-list/application-list-api/src/main/java/com/liferay/application/list/display/context/logic/PanelCategoryHelper.java

```diff
@@ -171,6 +172,13 @@ protected boolean hasPortlet(
 		return false;
 	}
 
+	public List<PanelCategory> getPanelCategories(
```

Comment:
Why is this method needed? You have access to `PanelCategoryRegistryUtil` where this method is called.